src/main/java/com/mabl/integration/jenkins/MablRestApiClientImpl.java

```diff
@@ -153,9 +167,31 @@ private CredentialsProvider getApiCredentialsProvider(
 
         provider.setCredentials(AuthScope.ANY, creds);
 
+        // Set proxy credentials if provided
+        if (proxy != null && !StringUtils.isBlank(proxy.getUserName())) {
+            final Credentials c = new UsernamePasswordCredentials(
+                    proxy.getUserName(),
+                    proxy.getPassword()
+            );
+            provider.setCredentials(new AuthScope(new HttpHost(proxy.name, proxy.port)), c);
+        }
+
         return provider;
     }
 
+    private CredentialsProvider getProxyCredentialsProvider(
+            final StandardUsernamePasswordCredentials credentials
+    ) {
+        final CredentialsProvider provider = new BasicCredentialsProvider();
+        final UsernamePasswordCredentials creds =
+                new UsernamePasswordCredentials(REST_API_USERNAME_PLACEHOLDER, restApiKey.getPlainText());
+
+        provider.setCredentials(AuthScope.ANY, creds);
```

Comment:
How's this work. Do more specific auth scopes (like proxy creds) override less specific cred scopes?

Reply:
Unused method. I left this in by mistake.